### ISSUE TITLE
Build failure: alot

### ISSUE THREAD

**Issue by brianmay** [NixOS/nixpkgs#358892]
## Steps To Reproduce

1. build *alot*

## Build log

```
nix-shell -p alot
this derivation will be built:
  /nix/store/piclm89wlmj6p77vzfi1iyndc97jkvic-alot-0.11.drv
building '/nix/store/piclm89wlmj6p77vzfi1iyndc97jkvic-alot-0.11.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
unpacking source archive /nix/store/vw065jnk2j1xcg89g6nzqxhv3y66kp6j-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/widgets/test_globals.py
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Setting SETUPTOOLS_SCM_PRETEND_VERSION to 0.11
Including all tracked files automatically
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
running egg_info
creating alot.egg-info
writing alot.egg-info/PKG-INFO
writing dependency_links to alot.egg-info/dependency_links.txt
writing entry points to alot.egg-info/entry_points.txt
writing requirements to alot.egg-info/requires.txt
writing top-level names to alot.egg-info/top_level.txt
writing manifest file 'alot.egg-info/SOURCES.txt'
reading manifest file 'alot.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'tests'
adding license file 'COPYING'
writing manifest file 'alot.egg-info/SOURCES.txt'
* Building wheel...
running bdist_wheel
running build
running build_py
creating build/lib/alot
copying alot/__init__.py -> build/lib/alot
copying alot/helper.py -> build/lib/alot
copying alot/account.py -> build/lib/alot
copying alot/__main__.py -> build/lib/alot
copying alot/ui.py -> build/lib/alot
copying alot/walker.py -> build/lib/alot
copying alot/crypto.py -> build/lib/alot
copying alot/errors.py -> build/lib/alot
creating build/lib/alot/widgets
copying alot/widgets/utils.py -> build/lib/alot/widgets
copying alot/widgets/namedqueries.py -> build/lib/alot/widgets
copying alot/widgets/__init__.py -> build/lib/alot/widgets
copying alot/widgets/bufferlist.py -> build/lib/alot/widgets
copying alot/widgets/search.py -> build/lib/alot/widgets
copying alot/widgets/thread.py -> build/lib/alot/widgets
copying alot/widgets/globals.py -> build/lib/alot/widgets
copying alot/widgets/ansi.py -> build/lib/alot/widgets
creating build/lib/alot/addressbook
copying alot/addressbook/abook.py -> build/lib/alot/addressbook
copying alot/addressbook/__init__.py -> build/lib/alot/addressbook
copying alot/addressbook/external.py -> build/lib/alot/addressbook
creating build/lib/alot/commands
copying alot/commands/utils.py -> build/lib/alot/commands
copying alot/commands/namedqueries.py -> build/lib/alot/commands
copying alot/commands/__init__.py -> build/lib/alot/commands
copying alot/commands/envelope.py -> build/lib/alot/commands
copying alot/commands/bufferlist.py -> build/lib/alot/commands
copying alot/commands/common.py -> build/lib/alot/commands
copying alot/commands/search.py -> build/lib/alot/commands
copying alot/commands/thread.py -> build/lib/alot/commands
copying alot/commands/taglist.py -> build/lib/alot/commands
copying alot/commands/globals.py -> build/lib/alot/commands
creating build/lib/alot/settings
copying alot/settings/utils.py -> build/lib/alot/settings
copying alot/settings/__init__.py -> build/lib/alot/settings
copying alot/settings/manager.py -> build/lib/alot/settings
copying alot/settings/const.py -> build/lib/alot/settings
copying alot/settings/theme.py -> build/lib/alot/settings
copying alot/settings/errors.py -> build/lib/alot/settings
creating build/lib/alot/utils
copying alot/utils/__init__.py -> build/lib/alot/utils
copying alot/utils/configobj.py -> build/lib/alot/utils
copying alot/utils/argparse.py -> build/lib/alot/utils
copying alot/utils/ansi.py -> build/lib/alot/utils
copying alot/utils/collections.py -> build/lib/alot/utils
copying alot/utils/cached_property.py -> build/lib/alot/utils
creating build/lib/alot/buffers
copying alot/buffers/namedqueries.py -> build/lib/alot/buffers
copying alot/buffers/__init__.py -> build/lib/alot/buffers
copying alot/buffers/envelope.py -> build/lib/alot/buffers
copying alot/buffers/bufferlist.py -> build/lib/alot/buffers
copying alot/buffers/search.py -> build/lib/alot/buffers
copying alot/buffers/thread.py -> build/lib/alot/buffers
copying alot/buffers/taglist.py -> build/lib/alot/buffers
copying alot/buffers/buffer.py -> build/lib/alot/buffers
creating build/lib/alot/completion
copying alot/completion/__init__.py -> build/lib/alot/completion
copying alot/completion/path.py -> build/lib/alot/completion
copying alot/completion/accounts.py -> build/lib/alot/completion
copying alot/completion/commandline.py -> build/lib/alot/completion
copying alot/completion/tag.py -> build/lib/alot/completion
copying alot/completion/abooks.py -> build/lib/alot/completion
copying alot/completion/cryptokey.py -> build/lib/alot/completion
copying alot/completion/commandname.py -> build/lib/alot/completion
copying alot/completion/stringlist.py -> build/lib/alot/completion
copying alot/completion/tags.py -> build/lib/alot/completion
copying alot/completion/namedquery.py -> build/lib/alot/completion
copying alot/completion/multipleselection.py -> build/lib/alot/completion
copying alot/completion/command.py -> build/lib/alot/completion
copying alot/completion/query.py -> build/lib/alot/completion
copying alot/completion/argparse.py -> build/lib/alot/completion
copying alot/completion/contacts.py -> build/lib/alot/completion
copying alot/completion/completer.py -> build/lib/alot/completion
creating build/lib/alot/db
copying alot/db/utils.py -> build/lib/alot/db
copying alot/db/attachment.py -> build/lib/alot/db
copying alot/db/__init__.py -> build/lib/alot/db
copying alot/db/envelope.py -> build/lib/alot/db
copying alot/db/manager.py -> build/lib/alot/db
copying alot/db/thread.py -> build/lib/alot/db
copying alot/db/errors.py -> build/lib/alot/db
copying alot/db/message.py -> build/lib/alot/db
running egg_info
writing alot.egg-info/PKG-INFO
writing dependency_links to alot.egg-info/dependency_links.txt
writing entry points to alot.egg-info/entry_points.txt
writing requirements to alot.egg-info/requires.txt
writing top-level names to alot.egg-info/top_level.txt
reading manifest file 'alot.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'tests'
adding license file 'COPYING'
writing manifest file 'alot.egg-info/SOURCES.txt'
creating build/lib/alot/defaults
copying alot/defaults/abook_contacts.spec -> build/lib/alot/defaults
copying alot/defaults/alot.rc.spec -> build/lib/alot/defaults
copying alot/defaults/default.bindings -> build/lib/alot/defaults
copying alot/defaults/default.theme -> build/lib/alot/defaults
copying alot/defaults/theme.spec -> build/lib/alot/defaults
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/alot
creating build/bdist.linux-x86_64/wheel/alot/widgets
copying build/lib/alot/widgets/utils.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/namedqueries.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/bufferlist.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/search.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/thread.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/globals.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
copying build/lib/alot/widgets/ansi.py -> build/bdist.linux-x86_64/wheel/./alot/widgets
creating build/bdist.linux-x86_64/wheel/alot/addressbook
copying build/lib/alot/addressbook/abook.py -> build/bdist.linux-x86_64/wheel/./alot/addressbook
copying build/lib/alot/addressbook/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/addressbook
copying build/lib/alot/addressbook/external.py -> build/bdist.linux-x86_64/wheel/./alot/addressbook
creating build/bdist.linux-x86_64/wheel/alot/commands
copying build/lib/alot/commands/utils.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/namedqueries.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/envelope.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/bufferlist.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/common.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/search.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/thread.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/taglist.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/commands/globals.py -> build/bdist.linux-x86_64/wheel/./alot/commands
copying build/lib/alot/__init__.py -> build/bdist.linux-x86_64/wheel/./alot
copying build/lib/alot/helper.py -> build/bdist.linux-x86_64/wheel/./alot
creating build/bdist.linux-x86_64/wheel/alot/settings
copying build/lib/alot/settings/utils.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/settings/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/settings/manager.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/settings/const.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/settings/theme.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/settings/errors.py -> build/bdist.linux-x86_64/wheel/./alot/settings
copying build/lib/alot/account.py -> build/bdist.linux-x86_64/wheel/./alot
creating build/bdist.linux-x86_64/wheel/alot/utils
copying build/lib/alot/utils/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/utils
copying build/lib/alot/utils/configobj.py -> build/bdist.linux-x86_64/wheel/./alot/utils
copying build/lib/alot/utils/argparse.py -> build/bdist.linux-x86_64/wheel/./alot/utils
copying build/lib/alot/utils/ansi.py -> build/bdist.linux-x86_64/wheel/./alot/utils
copying build/lib/alot/utils/collections.py -> build/bdist.linux-x86_64/wheel/./alot/utils
copying build/lib/alot/utils/cached_property.py -> build/bdist.linux-x86_64/wheel/./alot/utils
creating build/bdist.linux-x86_64/wheel/alot/buffers
copying build/lib/alot/buffers/namedqueries.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/envelope.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/bufferlist.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/search.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/thread.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/taglist.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/buffers/buffer.py -> build/bdist.linux-x86_64/wheel/./alot/buffers
copying build/lib/alot/__main__.py -> build/bdist.linux-x86_64/wheel/./alot
copying build/lib/alot/ui.py -> build/bdist.linux-x86_64/wheel/./alot
creating build/bdist.linux-x86_64/wheel/alot/defaults
copying build/lib/alot/defaults/default.bindings -> build/bdist.linux-x86_64/wheel/./alot/defaults
copying build/lib/alot/defaults/abook_contacts.spec -> build/bdist.linux-x86_64/wheel/./alot/defaults
copying build/lib/alot/defaults/theme.spec -> build/bdist.linux-x86_64/wheel/./alot/defaults
copying build/lib/alot/defaults/alot.rc.spec -> build/bdist.linux-x86_64/wheel/./alot/defaults
copying build/lib/alot/defaults/default.theme -> build/bdist.linux-x86_64/wheel/./alot/defaults
creating build/bdist.linux-x86_64/wheel/alot/completion
copying build/lib/alot/completion/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/path.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/accounts.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/commandline.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/tag.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/abooks.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/cryptokey.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/commandname.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/stringlist.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/tags.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/namedquery.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/multipleselection.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/command.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/query.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/argparse.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/contacts.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/completion/completer.py -> build/bdist.linux-x86_64/wheel/./alot/completion
copying build/lib/alot/walker.py -> build/bdist.linux-x86_64/wheel/./alot
copying build/lib/alot/crypto.py -> build/bdist.linux-x86_64/wheel/./alot
creating build/bdist.linux-x86_64/wheel/alot/db
copying build/lib/alot/db/utils.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/attachment.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/__init__.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/envelope.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/manager.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/thread.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/errors.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/db/message.py -> build/bdist.linux-x86_64/wheel/./alot/db
copying build/lib/alot/errors.py -> build/bdist.linux-x86_64/wheel/./alot
running install_egg_info
Copying alot.egg-info to build/bdist.linux-x86_64/wheel/./alot-0.11-py3.11.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/alot-0.11.dist-info/WHEEL
creating '/build/source/dist/.tmp-klewhs4j/alot-0.11-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'alot/__init__.py'
adding 'alot/__main__.py'
adding 'alot/account.py'
adding 'alot/crypto.py'
adding 'alot/errors.py'
adding 'alot/helper.py'
adding 'alot/ui.py'
adding 'alot/walker.py'
adding 'alot/addressbook/__init__.py'
adding 'alot/addressbook/abook.py'
adding 'alot/addressbook/external.py'
adding 'alot/buffers/__init__.py'
adding 'alot/buffers/buffer.py'
adding 'alot/buffers/bufferlist.py'
adding 'alot/buffers/envelope.py'
adding 'alot/buffers/namedqueries.py'
adding 'alot/buffers/search.py'
adding 'alot/buffers/taglist.py'
adding 'alot/buffers/thread.py'
adding 'alot/commands/__init__.py'
adding 'alot/commands/bufferlist.py'
adding 'alot/commands/common.py'
adding 'alot/commands/envelope.py'
adding 'alot/commands/globals.py'
adding 'alot/commands/namedqueries.py'
adding 'alot/commands/search.py'
adding 'alot/commands/taglist.py'
adding 'alot/commands/thread.py'
adding 'alot/commands/utils.py'
adding 'alot/completion/__init__.py'
adding 'alot/completion/abooks.py'
adding 'alot/completion/accounts.py'
adding 'alot/completion/argparse.py'
adding 'alot/completion/command.py'
adding 'alot/completion/commandline.py'
adding 'alot/completion/commandname.py'
adding 'alot/completion/completer.py'
adding 'alot/completion/contacts.py'
adding 'alot/completion/cryptokey.py'
adding 'alot/completion/multipleselection.py'
adding 'alot/completion/namedquery.py'
adding 'alot/completion/path.py'
adding 'alot/completion/query.py'
adding 'alot/completion/stringlist.py'
adding 'alot/completion/tag.py'
adding 'alot/completion/tags.py'
adding 'alot/db/__init__.py'
adding 'alot/db/attachment.py'
adding 'alot/db/envelope.py'
adding 'alot/db/errors.py'
adding 'alot/db/manager.py'
adding 'alot/db/message.py'
adding 'alot/db/thread.py'
adding 'alot/db/utils.py'
adding 'alot/defaults/abook_contacts.spec'
adding 'alot/defaults/alot.rc.spec'
adding 'alot/defaults/default.bindings'
adding 'alot/defaults/default.theme'
adding 'alot/defaults/theme.spec'
adding 'alot/settings/__init__.py'
adding 'alot/settings/const.py'
adding 'alot/settings/errors.py'
adding 'alot/settings/manager.py'
adding 'alot/settings/theme.py'
adding 'alot/settings/utils.py'
adding 'alot/utils/__init__.py'
adding 'alot/utils/ansi.py'
adding 'alot/utils/argparse.py'
adding 'alot/utils/cached_property.py'
adding 'alot/utils/collections.py'
adding 'alot/utils/configobj.py'
adding 'alot/widgets/__init__.py'
adding 'alot/widgets/ansi.py'
adding 'alot/widgets/bufferlist.py'
adding 'alot/widgets/globals.py'
adding 'alot/widgets/namedqueries.py'
adding 'alot/widgets/search.py'
adding 'alot/widgets/thread.py'
adding 'alot/widgets/utils.py'
adding 'alot-0.11.dist-info/COPYING'
adding 'alot-0.11.dist-info/METADATA'
adding 'alot-0.11.dist-info/WHEEL'
adding 'alot-0.11.dist-info/entry_points.txt'
adding 'alot-0.11.dist-info/top_level.txt'
adding 'alot-0.11.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built alot-0.11-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for alot-0.11-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: glibPreInstallPhase
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed alot-0.11-py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: glibPreFixupPhase
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11
checking for references to /build/ in /nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11...
patching script interpreter paths in /nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11
stripping (with command strip and flags -S -p) in  /nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11/lib /nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/lr1fxdlxghh2bny5ns84dgag22qz8vgm-alot-0.11-dist
checking for references to /build/ in /nix/store/lr1fxdlxghh2bny5ns84dgag22qz8vgm-alot-0.11-dist...
patching script interpreter paths in /nix/store/lr1fxdlxghh2bny5ns84dgag22qz8vgm-alot-0.11-dist
Rewriting #!/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/bin/python3.11 to #!/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10
wrapping `/nix/store/vavc8n3mkin38nj8c18b6f5j50gmfckv-alot-0.11/bin/alot'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Running phase: pythonRemoveBinBytecodePhase
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.3, pluggy-1.5.0
rootdir: /build/source
configfile: pyproject.toml
collected 40 items / 18 errors

==================================== ERRORS ====================================
_____________________ ERROR collecting extra/theme_test.py _____________________
ImportError while importing test module '/build/source/extra/theme_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
extra/theme_test.py:6: in <module>
    from alot.settings import theme
alot/settings/theme.py:6: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
_______________ ERROR collecting tests/commands/test_envelope.py _______________
ImportError while importing test module '/build/source/tests/commands/test_envelope.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/commands/test_envelope.py:26: in <module>
    from alot.commands import envelope
alot/commands/__init__.py:10: in <module>
    from ..settings.const import settings
alot/settings/const.py:4: in <module>
    from .manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
________________ ERROR collecting tests/commands/test_global.py ________________
ImportError while importing test module '/build/source/tests/commands/test_global.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/commands/test_global.py:24: in <module>
    from alot.commands import globals as g_commands
alot/commands/__init__.py:10: in <module>
    from ..settings.const import settings
alot/settings/const.py:4: in <module>
    from .manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
_________________ ERROR collecting tests/commands/test_init.py _________________
ImportError while importing test module '/build/source/tests/commands/test_init.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/commands/test_init.py:9: in <module>
    from alot import commands
alot/commands/__init__.py:10: in <module>
    from ..settings.const import settings
alot/settings/const.py:4: in <module>
    from .manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
________________ ERROR collecting tests/commands/test_thread.py ________________
ImportError while importing test module '/build/source/tests/commands/test_thread.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/commands/test_thread.py:11: in <module>
    from alot.commands import thread
alot/commands/__init__.py:10: in <module>
    from ..settings.const import settings
alot/settings/const.py:4: in <module>
    from .manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
__________________ ERROR collecting tests/db/test_envelope.py __________________
ImportError while importing test module '/build/source/tests/db/test_envelope.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/db/test_envelope.py:25: in <module>
    from alot.db import envelope
alot/db/__init__.py:4: in <module>
    from .thread import Thread
alot/db/thread.py:7: in <module>
    from .message import Message
alot/db/message.py:12: in <module>
    from . import utils
alot/db/utils.py:20: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
__________________ ERROR collecting tests/db/test_manager.py ___________________
ImportError while importing test module '/build/source/tests/db/test_manager.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/db/test_manager.py:13: in <module>
    from alot.db.manager import DBManager
alot/db/__init__.py:4: in <module>
    from .thread import Thread
alot/db/thread.py:7: in <module>
    from .message import Message
alot/db/message.py:12: in <module>
    from . import utils
alot/db/utils.py:20: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
__________________ ERROR collecting tests/db/test_message.py ___________________
ImportError while importing test module '/build/source/tests/db/test_message.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/db/test_message.py:21: in <module>
    from alot.db import message
alot/db/__init__.py:4: in <module>
    from .thread import Thread
alot/db/thread.py:7: in <module>
    from .message import Message
alot/db/message.py:12: in <module>
    from . import utils
alot/db/utils.py:20: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
___________________ ERROR collecting tests/db/test_thread.py ___________________
ImportError while importing test module '/build/source/tests/db/test_thread.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/db/test_thread.py:22: in <module>
    from alot.db import thread
alot/db/__init__.py:4: in <module>
    from .thread import Thread
alot/db/thread.py:7: in <module>
    from .message import Message
alot/db/message.py:12: in <module>
    from . import utils
alot/db/utils.py:20: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
___________________ ERROR collecting tests/db/test_utils.py ____________________
ImportError while importing test module '/build/source/tests/db/test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/db/test_utils.py:23: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
_______________ ERROR collecting tests/settings/test_manager.py ________________
ImportError while importing test module '/build/source/tests/settings/test_manager.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/settings/test_manager.py:14: in <module>
    from alot.settings.manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
________________ ERROR collecting tests/settings/test_theme.py _________________
ImportError while importing test module '/build/source/tests/settings/test_theme.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/settings/test_theme.py:7: in <module>
    from alot.settings import theme
alot/settings/theme.py:6: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
____________________ ERROR collecting tests/test_account.py ____________________
ImportError while importing test module '/build/source/tests/test_account.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_account.py:23: in <module>
    from . import utilities
tests/utilities.py:24: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
__________________ ERROR collecting tests/test_completion.py ___________________
ImportError while importing test module '/build/source/tests/test_completion.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_completion.py:10: in <module>
    from alot.completion.abooks import AbooksCompleter
alot/completion/abooks.py:7: in <module>
    from ..db.utils import formataddr
alot/db/__init__.py:4: in <module>
    from .thread import Thread
alot/db/thread.py:7: in <module>
    from .message import Message
alot/db/message.py:12: in <module>
    from . import utils
alot/db/utils.py:20: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
____________________ ERROR collecting tests/test_crypto.py _____________________
ImportError while importing test module '/build/source/tests/test_crypto.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_crypto.py:13: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
____________________ ERROR collecting tests/test_helper.py _____________________
ImportError while importing test module '/build/source/tests/test_helper.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_helper.py:29: in <module>
    from . import utilities
tests/utilities.py:24: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
________________ ERROR collecting tests/utils/test_configobj.py ________________
ImportError while importing test module '/build/source/tests/utils/test_configobj.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/utils/test_configobj.py:4: in <module>
    from alot.utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
________________ ERROR collecting tests/widgets/test_globals.py ________________
ImportError while importing test module '/build/source/tests/widgets/test_globals.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/widgets/test_globals.py:22: in <module>
    from alot.widgets import globals as globals_
alot/widgets/globals.py:13: in <module>
    from ..settings.const import settings
alot/settings/const.py:4: in <module>
    from .manager import SettingsManager
alot/settings/manager.py:17: in <module>
    from ..utils import configobj as checks
alot/utils/configobj.py:14: in <module>
    from .. import crypto
alot/crypto.py:5: in <module>
    import gpg
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py:123: in <module>
    from . import core
/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/core.py:10: in <module>
    from . import gpgme
E   ImportError: cannot import name 'gpgme' from partially initialized module 'gpg' (most likely due to a circular import) (/nix/store/vvfbch9bw910ya7cjdc6w46gf2k5y2cd-gpgme-1.23.2/lib/python3.11/site-packages/gpg/__init__.py)
=============================== warnings summary ===============================
../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.lcd_display is moved to urwid.display.lcd
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.lcd_display is moved to urwid.display.lcd
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.lcd_display is moved to urwid.display.lcd
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.lcd_display is moved to urwid.display.lcd
    return object.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.html_fragment is moved to urwid.display.html_fragment
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.html_fragment is moved to urwid.display.html_fragment
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.html_fragment is moved to urwid.display.html_fragment
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.html_fragment is moved to urwid.display.html_fragment
    return object.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.web_display is moved to urwid.display.web
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.web_display is moved to urwid.display.web
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.web_display is moved to urwid.display.web
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.web_display is moved to urwid.display.web
    return object.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.monitored_list is moved to urwid.widget.monitored_list
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.monitored_list is moved to urwid.widget.monitored_list
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.monitored_list is moved to urwid.widget.monitored_list
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.monitored_list is moved to urwid.widget.monitored_list
    return object.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.listbox is moved to urwid.widget.listbox
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.listbox is moved to urwid.widget.listbox
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.listbox is moved to urwid.widget.listbox
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.listbox is moved to urwid.widget.listbox
    return object.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:988: DeprecationWarning: urwid.treetools is moved to urwid.widget.treetools
    if ismodule(module) and hasattr(module, '__file__'):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:989: DeprecationWarning: urwid.treetools is moved to urwid.widget.treetools
    f = module.__file__

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:895: DeprecationWarning: urwid.treetools is moved to urwid.widget.treetools
    if getattr(object, '__file__', None):

../../nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896
  /nix/store/igyzv5c2v672mhmx1ha1i7062y6rggqs-python3-3.11.10/lib/python3.11/inspect.py:896: DeprecationWarning: urwid.treetools is moved to urwid.widget.treetools
    return object.__file__

alot/settings/manager.py:7
  /build/source/alot/settings/manager.py:7: DeprecationWarning: The mailcap module is deprecated and will be removed in Python 3.13. See the mimetypes module for an alternative.
    import mailcap

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR extra/theme_test.py
ERROR tests/commands/test_envelope.py
ERROR tests/commands/test_global.py
ERROR tests/commands/test_init.py
ERROR tests/commands/test_thread.py
ERROR tests/db/test_envelope.py
ERROR tests/db/test_manager.py
ERROR tests/db/test_message.py
ERROR tests/db/test_thread.py
ERROR tests/db/test_utils.py
ERROR tests/settings/test_manager.py
ERROR tests/settings/test_theme.py
ERROR tests/test_account.py
ERROR tests/test_completion.py
ERROR tests/test_crypto.py
ERROR tests/test_helper.py
ERROR tests/utils/test_configobj.py
ERROR tests/widgets/test_globals.py
!!!!!!!!!!!!!!!!!!! Interrupted: 18 errors during collection !!!!!!!!!!!!!!!!!!!
======================= 25 warnings, 18 errors in 0.65s ========================
error: builder for '/nix/store/piclm89wlmj6p77vzfi1iyndc97jkvic-alot-0.11.drv' failed with exit code 2;
       last 25 log lines:
       >   /build/source/alot/settings/manager.py:7: DeprecationWarning: The mailcap module is deprecated and will be removed in Python 3.13. See the mimetypes module for an alternative.
       >     import mailcap
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > ERROR extra/theme_test.py
       > ERROR tests/commands/test_envelope.py
       > ERROR tests/commands/test_global.py
       > ERROR tests/commands/test_init.py
       > ERROR tests/commands/test_thread.py
       > ERROR tests/db/test_envelope.py
       > ERROR tests/db/test_manager.py
       > ERROR tests/db/test_message.py
       > ERROR tests/db/test_thread.py
       > ERROR tests/db/test_utils.py
       > ERROR tests/settings/test_manager.py
       > ERROR tests/settings/test_theme.py
       > ERROR tests/test_account.py
       > ERROR tests/test_completion.py
       > ERROR tests/test_crypto.py
       > ERROR tests/test_helper.py
       > ERROR tests/utils/test_configobj.py
       > ERROR tests/widgets/test_globals.py
       > !!!!!!!!!!!!!!!!!!! Interrupted: 18 errors during collection !!!!!!!!!!!!!!!!!!!
       > ======================= 25 warnings, 18 errors in 0.65s ========================
       For full logs, run 'nix log /nix/store/piclm89wlmj6p77vzfi1iyndc97jkvic-alot-0.11.drv'.
```

<details>

<summary>Build Log</summary>

```
```

</details>

## Notify maintainers

@milibopp 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by milibopp** [comment]
This is essentially the same as #354166.
#354635 fixes this issue, but it still has not been merged into master yet.

**Comment by brianmay** [comment]
Oh, back port is here: #358568

Both PRs were merged, so should get to 24.11, just hasn't happened yet.

**Comment by dotlambda** [comment]
It will land once https://github.com/NixOS/nixpkgs/pull/369690 is merged.
